src/components/DocumentationTopic/DocumentationHero.vue

```diff
@@ -100,40 +98,36 @@ $doc-hero-icon-color: dark-color(fill-secondary) !default;
     top: 0;
   }
 
+  .icon {
+    position: absolute;
+    margin-top: 10px;
+    margin-right: $doc-hero-icon-spacing;
```

Comment:
Is the little offset on `top and `right`, intentional?

![image](https://user-images.githubusercontent.com/9863944/162372344-620718d7-6cf3-43d7-8545-7f6141403700.png)